app/models/mentor/request_lock.rb

```diff
@@ -3,4 +3,6 @@ class Mentor::RequestLock < ApplicationRecord
   belongs_to :locked_by, class_name: "User"
 
   scope :expired, -> { where("locked_until < ?", Time.current) }
+
+  MAX_LOCKS_PER_MENTOR = 2
```

Comment:
Is this a good location for the locks?

Reply:
Yeah, I think it's the right file. Pop the constant to the top of the class though. (Above the `belongs_to` calls)